lib/utility.js

```diff
@@ -1,10 +1,171 @@
 import Product from "../models/Product";
 
-// NOTE: variables and functions whose name starts with "_" are private and not to be exported
+// const marketUrl = "http://testcusa-env.afejb3cmmd.us-east-2.elasticbeanstalk.com/MarketExecute.php";
+// const loginUrl = "http://testcusa-env.afejb3cmmd.us-east-2.elasticbeanstalk.com/LoginExecute.php"
+const marketUrl = "https://madishare.com/MarketExecute.php";
+const loginUrl = "https://madishare.com/LoginExecute.php";
+let app = getApp();

```

Comment:
Looks like this is never used (only getUserId seems to be using it but it calls getApp() again)